frontend/src/main.ts

```diff
@@ -20,5 +21,6 @@ const app = createApp(App);
 
 app.use(router);
 app.use(pinia); // make sure this line is the one right above app.mount() for pinia to work properly in devtools
+app.use(VueCookies);
```

Comment:
Lil bro did you read the comment ^

Reply:
![image](https://github.com/tianhaoyao/gachasim/assets/32031491/5b93b1e6-d440-4630-8e0d-ca9f09d3c936)
still works

Reply:
can you remove the comment instead?